frameworks/helloworld/tests/test_tls.py

```diff
@@ -37,16 +37,7 @@
 
 
 @pytest.fixture(scope='module')
-def dcos_security_cli():
```

Comment:
We're dropping this becuase `sdk_security.create_service_account` does the same thing so its a dupe.